io/src/main/scala/com/velocidi/apso/io/FileDescriptor.scala

```diff
@@ -27,94 +29,111 @@ trait FileDescriptor {
 
   /** Returns true if the fd points to a directory
-    * @return true if the fd points to a directory
+    * @return
+    *   true if the fd points to a directory
     */
   def isDirectory: Boolean
 
   /** Lists the files in the current file descriptor directory
-    * @return a iterator of file descriptors
+    * @return
+    *   a iterator of file descriptors
     */
   def list: Iterator[FileDescriptor]
 
   /** Lists all files down the file hierarchy tree that whose relative path match the given prefix
-    * @param prefix the prefix to match given each file relative path to the current directory
-    * @return a iterator of file descriptors
+    * @param prefix
+    *   the prefix to match given each file relative path to the current directory
+    * @return
+    *   a iterator of file descriptors
     */
   def listAllFilesWithPrefix(prefix: String): Iterator[FileDescriptor]
 
   /** Returns the file descriptor `n` node up in the filesystem path.
-    * @param n the number of parent nodes to go back.
-    * @return the file descriptor `n` node up in the filesystem path.
+    * @param n
+    *   the number of parent nodes to go back.
+    * @return
+    *   the file descriptor `n` node up in the filesystem path.
     */
   def parent(n: Int = 1): FileDescriptor
 
   /** Adds a new child node to the filesystem path.
-    * @param name the node name.
-    * @return the new file descriptor with the updated path
+    * @param name
+    *   the node name.
+    * @return
+    *   the new file descriptor with the updated path
     */
   def child(name: String): FileDescriptor
 
   /** Adds a new child node to the filesystem path.
-    * @param name the node name.
-    * @return the new file descriptor with the updated path
+    * @param name
+    *   the node name.
+    * @return
+    *   the new file descriptor with the updated path
     */
   def /(name: String): FileDescriptor = child(name)
 
   /** Adds multiple new child nodes to the filesystem path.
-    * @param names the node names to add.
-    * @return the new file descriptor with the updated path
+    * @param names
+    *   the node names to add.
+    * @return
+    *   the new file descriptor with the updated path
     */
   def children(names: String*): FileDescriptor =
     names.foldLeft(this)((acc, c) => acc.child(c))
 
-  /** Changes the path of the file descriptor using unix's cd syntax related to the current directory.
-    * Ex: Go back to directories: ../..
-    *     Go to child directories: foo/bar
-    *     Go to sibling directory: ../foo
-    * Absolute path changes are not supported.
-    * @param pathString the cd command
-    * @return the new file descriptor with the updated path
+  /** Changes the path of the file descriptor using unix's cd syntax related to the current directory. Ex: Go back to
+    * directories: ../.. Go to child directories: foo/bar Go to sibling directory: ../foo Absolute path changes are not
+    * supported.
```

Comment:
Can we replace these examples with a list to avoid reformat?

```suggestion
  /** Changes the path of the file descriptor using unix's cd syntax related to the current directory.
    *
    * Ex:
    *   - Go back to directories: ../..
    *   - Go to child directories: foo/bar
    *   - Go to sibling directory: ../foo
    *
    * Absolute path changes are not supported.
```